storybook/addons/SuspenseDecorator.js

```diff
@@ -0,0 +1,3 @@
+import React, { Suspense } from 'react';
+
+export default storyFn => <Suspense fallback={<div>loading</div>}>{storyFn()}</Suspense>;
```

Comment:
what is this for exactly?

Reply:
also you can use a fragment instead of div

Reply:
> what is this for exactly?

I was getting suspense err when rendering the components, this fixed it. 💃 
Now I'll dig why there was the error.

Reply:
```
Wrapper suspended while rendering, but no fallback UI was specified. Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display. in Wrapper (at story.js:6) in I18nextProvider (at LocaleDecorator.js:6) in InnerComponent (created by HistoryWatcher) in HistoryWatcher (created by Route) in Route (created by StoryRouter) in Router (created by MemoryRouter) in MemoryRouter (created by StoryRouter) in StoryRouter
```

Reply:
@Kamahl19 it's required for the `withTranslations` HoC.
I guess I will wrap just the `LocaleDecorator` with the Suspense then...